stable/spinnaker/values.yaml

```diff
@@ -31,14 +31,14 @@ mail:
 
 # Images for each component
 images:
```

Comment:
It's kinda ok then. I did not realized it's separate commits. Let's just not squash them.